interface.html

```diff
@@ -49,7 +49,7 @@
           <div class="col-sm-8">
             <label for="page" class="select-proxy-display">
               <select name="link_screen" id="page" data-label="select" class="hidden-select form-control">
-                <option selected value="none">Select a screen</option>
+                <option value="defaultSelect">Select a screen</option>
```

Comment:
```suggestion
                <option value="none">Select a screen</option>
```

Reply:
It will work with  value="none". I just changed it to be more clear, I will change it back then.

Reply:
fixed